### PR TITLE
Marketplace Add: Q1 Partner Release

### PR DESCRIPTION
**Pull request by hmorris3293**
## Description
This PR adds the fakespec & svgs for our Q1 Partner release for Marketplace

## Type of Change
 * Non breaking change ('update', 'change')
 
## Note to Reviewers
Full release for prod will need to be coordinated with Marketing.

### CODE REVIEW

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 1]
```diff
@@ -1126,5 +1164,43 @@ export const oneClickApps: OCA[] = [
...
+      {
+        title: 'https://github.com/SelfhostedPro/Yacht/',
+        href: 'https://github.com/SelfhostedPro/Yacht/'
+      }
+    ],
+    logo_url: 'assets/yacht_color.svg'
+  },
+  {
+    name: 'Zabbix',
+    description: `Host multiple sites on a single server while managing apps, firewall, databases, backups, system users, cron jobs, SSL and email–  all in an intuitive interface.`,
```
Comment (Jskobos) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:1189]: This is the same description as serverwand above.

File: packages/manager/src/features/OneClickApps/FakeSpec.ts [hunk 2]
```diff
@@ -118,6 +118,25 @@ export const oneClickApps: OCA[] = [
     ],
     logo_url: 'assets/csgo_color.svg'
   },
+  {
+    name: 'Cyberpanel',
```
Comment (Jskobos) [inline packages/manager/src/features/OneClickApps/FakeSpec.ts:122]: Needs to be capitalized.

**Review by Jskobos — CHANGES_REQUESTED** [review]
What about Tomcat and Aberrit Node.js? There are scripts for them but no info here. Have they been pushed to the next release?

File: packages/manager/src/features/OneClickApps/FakeSpec.ts (hunk 1 above)
Comment (hmorris3293) [reply packages/manager/src/features/OneClickApps/FakeSpec.ts]: fixed!

File: packages/manager/src/features/OneClickApps/FakeSpec.ts (hunk 2 above)
Comment (hmorris3293) [reply packages/manager/src/features/OneClickApps/FakeSpec.ts]: fixed!

**Comment by hmorris3293** [comment]
@Jskobos Aberrit NodeJS & Tomcat did not make it into this release, so it is just the 4 apps

**Review by Jskobos — APPROVED** [review]

**Comment by Jskobos** [comment]
The new apps have color icons used in the card view, they should be B&W

**Review by tiffwong — APPROVED** [review]

**Pull request merged**